### PR TITLE
[ADD] Adding Main Adresse and Main Email in the filter for the

### PR DESCRIPTION
**Pull request by EcoloDevelopper**
Assemblies/Instances virtual models; refs #29736

### CODE REVIEW

File: odoo_addons/mozaik_communication/views/virtual_models_view.xml [hunk 1]
```diff
@@ -823,7 +823,12 @@
                     <field name="int_power_level_id" string="Internal Power Level" />
 
                     <field name="competencies_m2m_ids" widget="many2one" />
-
+					<filter string="Main Address" name="main_postal"
```
Comment (Olivier-LAURENT) [inline odoo_addons/mozaik_communication/views/virtual_models_view.xml]: Special indentation here ! Please convert tabs to spaces in your editor settings.

File: odoo_addons/mozaik_communication/virtual_models.py [hunk 2]
```diff
@@ -855,10 +855,14 @@ class virtual_assembly_instance(orm.Model):
...
             'sta.assembly.category', 'State Assembly Category'),
-
+        
+        'main_postal': fields.boolean('Main Address'),
+        
         'postal_vip': fields.boolean('VIP Address'),
         'postal_unauthorized': fields.boolean('Unauthorized Address'),
 
+        'main_email': fields.boolean('Main Email'),
+
```
Comment (Olivier-LAURENT) [inline odoo_addons/mozaik_communication/virtual_models.py]: remove this blank line

File: odoo_addons/mozaik_communication/virtual_models.py [hunk 3]
```diff
@@ -855,10 +855,14 @@ class virtual_assembly_instance(orm.Model):
             'ext.assembly.category', 'External Assembly Category'),
         'sta_category_assembly_id': fields.many2one(
             'sta.assembly.category', 'State Assembly Category'),
-
+        
+        'main_postal': fields.boolean('Main Address'),
+
```
Comment (Olivier-LAURENT) [inline odoo_addons/mozaik_communication/virtual_models.py]: remove this blank line

File: odoo_addons/mozaik_communication/virtual_models.py [hunk 4]
```diff
@@ -855,10 +855,14 @@ class virtual_assembly_instance(orm.Model):
             'ext.assembly.category', 'External Assembly Category'),
         'sta_category_assembly_id': fields.many2one(
             'sta.assembly.category', 'State Assembly Category'),
-
+
```
Comment (Olivier-LAURENT) [inline odoo_addons/mozaik_communication/virtual_models.py]: this blank line seems to be modified !? Did you add blanks ? Please remove them ?

**Review by Olivier-LAURENT — COMMENTED** [review]
Some cosmetic remarks...

**Comment by Olivier-LAURENT** [comment]
@EcoloDevelopper 
Please make a rebase on 8.0 + push -f to avoid noise about my last commits

**Comment by EcoloDevelopper** [comment]
Rebase done

**Comment by Olivier-LAURENT** [comment]
@EcoloDevelopper 
I have a doubt about your rebase otherwise all changes in the `Files changed` tab should be yours... and it's not the case. Are you sure you made a `push -f` after having made a `rebase remotes/origin/8.0` after having made a `fetch -a` ??

**Comment by Olivier-LAURENT** [comment]
@EcoloDevelopper 
You forget to translate your 2 new filters.

**Pull request merged**